pkg/lib/cloudstorage/minio.go

```diff
@@ -0,0 +1,108 @@
+package cloudstorage
```

Comment:
I thought we are integrating minio same as S3, because it is S3 compatible 🤔

Reply:
Theoretically, yes. But MinIO docs does not cover how to use the AWS SDK with it. So I install the MinIO SDK instead.